decode_morse_code.rb

```diff
@@ -0,0 +1,22 @@
+CODE = { ".-" => "A", "-..." => "B", "-.-." => "C", "-.." => "D", "." => "E",
+  "..-." => "F", "--." => "G", "...." => "H", ".." => "I", ".---" => "J", "-.-" => "K",
+  ".-.." => "L", "--" => "M", "-." => "N", "---" => "O", ".--." => "P", "--.-" => "Q",
+  ".-." => "R", "..." => "S", "-" => "T", "..-" => "U", "...-" => "V", ".--" => "W",
+  "-..-" => "X", "-.--" => "Y", "--.." => "Z", "-----" => "0", ".----" => "1",
+  "..---" => "2", "...--" => "3", "....-" => "4", "....." => "5", "-...." => "6",
+  "--..." => "7", "---.." => "8", "----." => "9" }
+
+def decode(message)
+  words = message.split('   ')
+  decoded_words = words.map do |word|
```

Comment:
Good job so far, but in the project requirements you were asked to implement 3 methods :
**Project requirements
Create a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase (e.g. decode_char(".-") returns "A").

Create a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. decode_word("-- -.--") returns "MY").

Create a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word will be separated by 3 spaces (e.g.

** 

and that's a good practice for reusability an simplicity of code